### ISSUE TITLE
iOS 15: tab bar black in light mode, and "My Store" menu no more visible during the first loading

### ISSUE THREAD

**Issue by pmusolino** [woocommerce/woocommerce-ios#5018]
I discovered this bug that is currently happening just on iOS 15.0 (I tested it in the simulator, on iPhone 13 Pro).

During the first loading of the dashboard, if the iPhone is in light mode, the tab bar will become black, and the "My Store" menu is invisible.

This happens just if you build the app with Xcode 13.

<img src="https://user-images.githubusercontent.com/495617/133990834-f63a8684-7054-4240-b685-ee90435c7d9d.png" width=300 />

**Comment by jaclync** [comment]
This also happens in dark mode, just that it's not as obvious as in light mode. It seems to happen when each tab is in loading state (screenshots below). I saw a similar issue in WPiOS https://github.com/wordpress-mobile/WordPress-iOS/issues/17172 and @momo-ozawa shared a fix to set the tabbar's `scrollEdgeAppearance` (new in iOS 15) along with `standardAppearance`. I tried it in WCiOS by adding the following diffs

```swift
        if #available(iOS 15.0, *) {
            appearance.scrollEdgeAppearance = appearance.standardAppearance
        }
```

under

https://github.com/woocommerce/woocommerce-ios/blob/fb25492561c0d5bc16ab148b314969583880b886/WooCommerce/Classes/Extensions/UITabBar+Appearance.swift#L17



and it seems to work 😄 But we have to wait for Xcode 13 to put in this fix since `scrollEdgeAppearance` is iOS 15+, after Platform9 thinks it's ready with CI and tooling.

loading state | loaded
-- | --
<img src="https://user-images.githubusercontent.com/1945542/134625318-c3b49280-9720-49d2-8e85-b068f82176da.png" width="300" /> | <img src="https://user-images.githubusercontent.com/1945542/134625250-a9506751-03b4-44e3-bb33-ba6edc59cda1.png" width="300" />

**Comment by pmusolino** [comment]
Sounds good. Thanks for investigating on this @jaclync 🙌 💯

**Comment by jaclync** [comment]
Kudos to @momo-ozawa for investigating the original issue in WPiOS! 🥇

**Comment by pmusolino** [comment]
So, thank you both! 👯 @jaclync @momo-ozawa

**Comment by shiki** [comment]
@pmusolino Is this supposed to be closed by #5312?

**Comment by jaclync** [comment]
I think this issue wasn't closed by #5312 because it wasn't targeting `develop` at the time. But the PR description says `Fixes #5018` so I assume Paolo meant to close this issue 🙂

**Comment by pmusolino** [comment]
As said by @jaclync the issue wasn't closed because it wasn't targeting `develop` at the time, but yes it's fixed :) I'm going to close it right now.
